app/views/data_services/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="govuk-heading-xl">Find data</h1>
+
+<% @organisations.each do |organisation| %>
+    <h2 class="govuk-heading-l"><%= organisation.name %></h2>
+
+    <ul class="govuk-list govuk-list--bullet">
+        <% organisation.data_services.each do |data_service| %>
```

Comment:
Nitpick: I would extract this into a partial and use `collection`. I like to always represent the same informations in the same way, if we decide that data services in a list have this representation it will help first of all the users.

Call to render could be:
```
<%= render partial: "data_service", collection: organisation.data_services %>
```